### PR TITLE
SVCPLAN-2588: add tempsetDNS postscript

### PR DESCRIPTION
**Pull request by jakerundall**
The tempsetDNS postscript is used to configure NCSA DNS in /etc/resolv.conf. This is needed after running confignetwork with the -s flag to set interfaces to static. When this is run Network Manager clobbers /etc/resolv.conf and we've been unable to figure out how to get xCAT to avoid/correct this "natively". So this would be run after 'confignetwork -s' and prior to anything that requires functioning DNS, e.g., Puppet.

### CODE REVIEW

**Comment by jakerundall** [comment]
Note that I added the 2nd NCSA DNS server into the list here, just in case.

File: postscripts/simple_fw
Comment (billglick) [inline postscripts/simple_fw:1]: Can this file be removed?

**Review by billglick — APPROVED** [review]
This generally looks good.

It does appear to add an empty `postscripts/simple_fw` that I think should be removed.

I ran into this issue when setting up the new npcf-influxdb server with `asd-prov01`, and created a specific custom postscript for it, but this is great to generalize it.

File: postscripts/simple_fw
Comment (jakerundall) [reply postscripts/simple_fw:1]: This file is already in main but my branch adds execute permissions to it (to match the other postscripts in this repo).

**Pull request merged**